PWGCF/Flow/Tasks/FlowTask.cxx

```diff
@@ -302,6 +318,16 @@ struct FlowTask {
     corrconfigs.push_back(fGFW->GetCorrelatorConfig("refN10 {3 2} refP10 {-3 -2}", "Ch10Gap3232", kFALSE));
     corrconfigs.push_back(fGFW->GetCorrelatorConfig("refN10 {4 2} refP10 {-4 -2}", "Ch10Gap4242", kFALSE));
     corrconfigs.push_back(fGFW->GetCorrelatorConfig("refN10 {2 2} refP10 {-2 -2}", "Ch10Gap24", kFALSE));
+    if (!UserDefineGFWCorr.empty() && !UserDefineGFWName.empty()) {
+      LOGF(info, "User adding GFW CorrelatorConfig:");
+      // attentaion: here we follow the index of cfgUserDefineGFWCorr
+      for (int i = 0; i < UserDefineGFWCorr.size(); i++) {
+        if (i >= UserDefineGFWName.size())
+          continue;
```

Comment:
Shouldn't this condition give a `fatal` to the user telling him that something has been wrongly configured?

Reply:
sure, I will add fatal signal here